common/sign/v1/iss.c.inc

```diff
@@ -105,7 +121,7 @@ int _ISS_PREFIX(sig_digest)(trit_t *const dig, trit_t *const hash, trit_t *sig,
     }
   }
   _HASH_PREFIX(absorb)(state, sig_start, sig_len);
-  _HASH_PREFIX(squeeze)(state, dig, HASH_LENGTH);
+  //_HASH_PREFIX(squeeze)(state, dig, HASH_LENGTH);
```

Comment:
I'm not 100% sure about commenting that out as sig_digest is not tested.
Should this squeeze really be here ?
I haven't been able to complete an address from a signature digest with it being here.